plugins/moldyn/src/io/TestSpheresDataSource.cpp

```diff
@@ -93,19 +94,23 @@ void TestSpheresDataSource::loadFrame(AnimDataModule::Frame* frame, unsigned int
     auto frameCount = this->numFramesSlot.Param<core::param::IntParam>()->Value();
     auto sphereCount = this->numSpheresSlot.Param<core::param::IntParam>()->Value();
     frm->data = new float[8 * sphereCount];
+    std::mt19937 twister;
+    auto new_rand = [&twister]() -> int32_t { return twister() & RAND_MAX; };
+    auto new_seed = [&twister](uint32_t i) { twister.seed(i); };
     for (unsigned int i = 0; i < sphereCount; i++) {
         vislib::math::ShallowVector<float, 3> pos(&frm->data[i * 8]);
-        ::srand(i); // stablize values for particles
+        new_seed(i); // stablize values for particles
```

Comment:
http://www0.cs.ucl.ac.uk/staff/d.jones/GoodPracticeRNG.pdf "Avoid setting the seeds to zero or small numbers in general – try to choose large “complex” seed values "